pandas/core/indexing.py

```diff
@@ -662,7 +662,9 @@ def _ensure_listlike_indexer(self, key, axis=None, value=None):
         if self.ndim != 2:
             return
 
-        if isinstance(key, tuple) and not isinstance(self.obj.index, ABCMultiIndex):
+        if isinstance(key, tuple) and (
```

Comment:
don't we need len(key) as a condition?

Reply:
wow this is a specific condition (i know i asked you to update). umm anyway we can be more general here?

Reply:
Yeah, was not happy with that either. This is better and does the job now.